pay-api/src/pay_api/models/routing_slip.py

```diff
@@ -137,8 +138,11 @@ def _add_date_filter(cls, query, search_filter):
 
             created_from = created_from.replace(hour=0, minute=0, second=0, microsecond=0).astimezone(tz_local)
             created_to = created_to.replace(hour=23, minute=59, second=59, microsecond=999999).astimezone(tz_local)
+
+            target = getattr(RoutingSlip, get_str_by_path(search_filter, 'dateFilter/target') or 'routing_slip_date')
```

Comment:
good code..
may be call it as target_date Or add a one liner about this..its not very straightforward logic since we have two close variable to search on ...